src/main/routes/task-list.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express';
+import { SubTaskListBuilder } from '../app/case/util/subTaskListBuilder';
+
+const router = express.Router();
+
+router.get('/task-list', (req, res) => {
+  res.render('task-list', {
+    tasks: {
+      applicantDetails: SubTaskListBuilder.buildApplicationDetailsSubTasks(),
+      yourDetails: SubTaskListBuilder.buildYourDetailsSubTasks(),
+      childDetails: SubTaskListBuilder.buildChildDetailsSubTasks(),
```

Comment:
We have to plan on how to handle for example when childDetails is undefined. I guess we can discuss this at later point or maybe we can do something now, don't know... to discuss.